python/requirements/ml/dl-cpu-requirements.txt

```diff
@@ -4,6 +4,7 @@
 tensorflow==2.11.0; sys_platform != 'darwin' or platform_machine != 'arm64'
 tensorflow-macos==2.11.0; sys_platform == 'darwin' and platform_machine == 'arm64'
 tensorflow-probability==0.19.0
+tensorflow-io-gcs-filesystem==0.31.0
```

Comment:
@matthewdeng fyi, I'm pinning this package so that it is compatible with windows. I also notice that pypi actually recommend to use this version for tensorflow 2.11

<img width="800" alt="Screenshot 2023-12-13 at 9 37 42 PM" src="https://github.com/ray-project/ray/assets/128072568/4b0bdfce-c1cc-4cdc-9128-e433e9c853c8">